### DISCUSSION TITLE
Split BT-s?

### DISCUSSION THREAD

**Discussion by ChristianeCPeters** [OP-TED/eForms-SDK#394] [Q&A] [answered]
In the CELEX -no matter what version- theBT-s are groupt within each BG and the colums give Info in which notive Type the BG/BT is Optional, Conditionaly Mandatory, Existing Mandatory or Mandatory. So far so good. 
None of the BTs or BGs are devided in the CELEX. In the SDK -also different Versions-   however, ther are Rules for different "Parts" of BTs. Eg. BT-09 a and BT-09 b or BT-712 a and BT-712b. Where do theses divisions come from and where can Info on the parted BTs can be retrieved?

**Comment by bertrand-lorentz** [comment, marked as answer]
Please see this page for explanations on why there sometimes are several fields for one BT:
https://docs.ted.europa.eu/eforms/latest/fields/index.html

As indicated there, for each field we indicate in `fields.json` the identifier of the BT to which the field corresponds.

As this is rather a question, I'm converting this to a "discussion".
